js/packages/web/locale/en.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Address to mint edition to": "Address to mint edition to",
+  "Buy exclusive {name} NFTs.": "Buy exclusive {name} NFTs.",
+  "Congratulations": "Congratulations",
+  "Got it": "Got it",
+  "Mint": "Mint",
+  "Mint to": "Mint to",
+  "My Items": "My Items",
+  "New editions have been minted please view your NFTs in": "New editions have been minted please view your NFTs in",
+  "Number of editions to mint": "Number of editions to mint",
+  "The amazing world of {name}": "The amazing world of {name}",
+  "Total cost": "Total cost"
+}
```

Comment:
```suggestion
}

```

Reply:
same here

Reply:
ja.json and en.json is a file generated automatically by lingui.
So I will see if I can add a newline.

Reply:
Ah right I didn't know they re autogenerated, thanks

Reply:
I recreated the json by lingui and it added a newline at the end.